.github/workflows/changelog-assembly.yml

```diff
@@ -32,6 +32,8 @@ jobs:
           crate: aeruginous
 
       - run: |
+          aeruginous comment-changes \
+            -b -d : -f ron -k -o changelog.d/ -S "$(cat .version)"
```

Comment:
This is the new CHANGELOG fragment creation logic.  In contrast to the old one, obviously the `-S` flag is new to make Aeruginous stop at the last release commit.  Furthermore, I needed to drop the `-C` flag as it would also have harvested Dependabot update commits which might have led to unnecessary entries in the rendered fragment.  The downside of doing so is that now, the PR titles **must** match the Keep a Changelog category pattern.  I did not find any solution to avoid this tradeoff, yet, I am sorry.

Reply:
I just wanted to make sure, does the new workflows really checks the PR titles or the commits messages and description?

Reply:
The logic checks the body of the commit messages.  When merging a PR with creating a merge commit, GitHub will add the PR title to the merge commit's body.

Reply:
Thanks for the clarification, I needed to know that so I know to copy the PR title to the commit in case I used the squash and merge approach or even pushed a commit directly to the main branch